src/lib/DataTable/DataTableInterfaces.ts

```diff
@@ -1,9 +1,12 @@
+/* eslint-disable max-lines */
```

Comment:
can you may put something like this? like this we don't have it completely open and need to reconsider again if we wanna split it if we add more code. 

/* eslint max-lines: ["error", 220] */ // comment why :D